src/state/apis/zapper/zapperApi.ts

```diff
@@ -319,6 +319,9 @@ export const zapper = createApi({
   endpoints: build => ({
     getZapperUniV2PoolAssetIds: build.query<GetZapperUniV2PoolAssetIdsOutput, void>({
       queryFn: async (_, { dispatch }) => {
+        // We already have the static assets list in the store, however we need to also fetch it here for 2 reasons
+        // 1. We still need to fetch the raw data from Zapper, which we can't store dynamically because asset ratios and APY changes
+        // 2. We have no notion of asset "tags" i.e the notion that these assets are UNI-V2 pools
```

Comment:
Technically, we could remove the `await` here if we filtered down by `UNI-V2` symbol. Not a fan about these heuristics however, and we still need to fetch this to get the *akschual* pool data (as opposed to assets data), so...